### PR TITLE
refactor: clean up code

### PR DESCRIPTION
**Pull request by deliaBlue**

### CODE REVIEW

**Review by uniqueg — CHANGES_REQUESTED** [review]
I know I kinda okayed to have this PR like that, but I thought it would really be a cleanup. I have identified four different PRs in this one:

- Clean up
- Temporarily remove isomiR quantification
- Modify test files
- Snakemake linting/formatting

Plus the changes to the quantification script that I don't agree with.

I know that in principle you could make this four different commits in a single PR, but I don't want it like that, for 2 reasons:
- It will be tricky to keep the commits really focused on these changes, and for every requested change, it may be painful to assign those changes to the corresponding commit - it will just turn into a mess
- It's a pain in the ass to review - especially if I want to make sure that every commit contains everything for each of these different topics, and nothing on top of it.

File: workflow/rules/map.smk
Comment (deliaBlue) [reply workflow/rules/map.smk:676]: Upstream everything is "grouped" in such a way that first we get genome processes done and then the transcriptome ones. I thought that having this swapped makes more sense as first there will be the last genome modifications and then the transcriptome one before merging both. I can reorder it for this PR but if you agree, I'll leave it like this for the snakemake formatting PR.

**Comment by uniqueg** [comment]
I got a notification about changes here, but can't really spot what it is. Please tag me again for review when you are ready

**Comment by deliaBlue** [comment]
I will wait for the CI to pass on the other PR and then merge it with the this branch and make the last change on this branch (remove commented isomiRs rules on the `quanitfy.smk`) because I revert the files but only some of them did and it is being a little messy to reorganize this branch again.

**Comment by uniqueg** [comment]
Sure, just flag me for review when you're done :)

**Review by uniqueg — APPROVED** [review]

**Pull request merged**
